airflow/api_connexion/endpoints/dag_run_endpoint.py

```diff
@@ -70,10 +92,14 @@ def get_dag_runs(session, dag_id, start_date_gte=None, start_date_lte=None,
 
     # filter execution date
     if execution_date_gte:
-        query = query.filter(DagRun.execution_date >= conn_parse_datetime(execution_date_gte))
+        query = query.filter(
+            DagRun.execution_date >= conn_parse_datetime(execution_date_gte)
```

Comment:
It looks to me, conflicts was not resolved well on this endpoint. Can you check the one on master. It uses format_parameters to parse date instead of conn_parse_datetime

Reply:
I haven't rebased it yet, this will be fixed once I do that.

Reply:
Oh, that's nice. Brilliant idea. Haha. I use to rebase everytime and having issues. Will start using this method. Thanks @Omairk :)

Reply:
Can you do a rebase? I would like to slowly finish this change.